api-spec/wfs-stac.md

```diff
@@ -47,8 +48,9 @@ GET /collections
 
 Lists the collections of data on the server that can be queried ([7.11](https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_feature_collections_metadata)), 
 and each describes basic information about the geospatial data collection, like its name and description, as well as the 
-spatial and temporal extents of all the data contained. A STAC search extension would only query those collections which
-have data that validates as STAC `Items` - with a datetime field and references to assets. But a STAC can live alongside
+spatial and temporal extents of all the data contained. The collections returned are compliant to both WFS Collections and 
+[STAC collections](../collections-spec/README.md). A STAC search extension would only query those collections which
```

Comment:
I would ideally like a bit more discussion here, reiterating the point of the STAC collections, and contrasting them against WFS collections, perhaps mentioning any additional alignment we'd see in the future, etc. Not required to merge this by any means, but definitely a nice to have.

Reply:
Should be added at some point, but don't feel that I can really make a good proposal for this one now. Feel free to make a PR for it?!